app/views/expenditures/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @expenditure %>
+
+<div>
+  <%= link_to "Edit this expenditure", edit_expenditure_path(@expenditure) %> |
+  <%= link_to "Back to expenditures", expenditures_path %>
+
+  <%= button_to "Destroy this expenditure", @expenditure, method: :delete %>
+</div>
```

Comment:
As stated by the previous CR, kindly ensure instead of the back to expenditure button in the header, you have an arrow-back Icon as seen on the original design for a better UI & UX 😉 

Original Design                                        |                      Your Design
![Screenshot from 2023-08-18 18-16-38](https://github.com/prius2055/rails-budget-app/assets/44624138/a8996d86-b7d3-47f1-8deb-350e878c4b82)  |  ![Screenshot from 2023-08-18 18-14-04](https://github.com/prius2055/rails-budget-app/assets/44624138/61416b7f-ef0e-4248-aba9-fc17351aa888)

Reply:
This was an omission. But has been fixed. Thank you